cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -100,3 +102,14 @@ func makeModelKeyValue(agentTags map[string]string) []model.KeyValue {
 
 	return tags
 }
+
+func (r *Reporter) Retryable(err error) bool {
+	state := status.Convert(err)
+	switch state.Code() {
+	case codes.DeadlineExceeded:
+		fallthrough
```

Comment:
can we just return? easier to read

Reply:
There's actually more of them, at least Unknown as well (on some runs I can see that from timeouts). But yeah, I can do return from them as well  although to me that makes it more difficult to read ;) (Go is probably the only language with this type of switch..)